src/usb/uf2/ghostfat.c

```diff
@@ -390,7 +390,12 @@ int write_block (uint32_t block_no, uint8_t *data, WriteState *state)
 
   switch ( bl->familyID )
   {
-    case CFG_UF2_FAMILY_APP_ID:
+
+#if defined(CFG_UF2_BOARD_APP_ID) && CFG_UF2_BOARD_APP_ID
```

Comment:
please remove this #if as well

Reply:
Well, I **_could_** remove that `#if`/`#endif` pair.
However, I don't recommend doing so, as it will break the build for the `832 board, which doesn't define that symbol.

Reply:
no it wont' this file won't be compiled with 832.The previous issue exist because the define in the header file, which actually shouldn't be included by 832. Though I will fix that later on as mentioned above.

Reply:
_mea culpa!_  you're right ... I'm sorry.  Pushed the change.